src/modules/users/users.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { UsersService } from './users.service';
+import { AuthService } from '../auth/auth.service';
+import { CreateUserDto } from './dto/create-user.dto';
+import { LoginDto } from './dto/login.dto';
+
+@Controller('users')
+export class UsersController {
+    constructor(
+        private usersService: UsersService,
+        private authService: AuthService
+    ) {}
+
+    @Post()
+    async createUser(@Body() createUserDto: CreateUserDto) {
```

Comment:
```suggestion
    async create(@Body() createUserDto: CreateUserDto) {
```

đặt tên như này là đủ dùng chưa em nhỉ, có bị nhầm lẫn với chỗ khác ko?